src/libcore/cmp.rs

```diff
@@ -35,7 +35,7 @@ use self::Ordering::*;
 ///
 /// This trait allows for partial equality, for types that do not have a full
 /// equivalence relation. For example, in floating point numbers `NaN != NaN`,
-/// so floating point types implement `PartialEq` but not [`Eq`].
+/// so floating point types implement `PartialEq` but not [`Eq`](Eq).
```

Comment:
Better leaving it like it was (``[`Eq`]``). Just removing the ``/// [`Eq`]: Eq`` line should be enough.

Reply:
Doing so makes the `` [`Eq`] `` link point to ``[`eq`]`` (which is a bug in `rustdoc` I think, but something we have to get around for now).